styles/style.css

```diff
@@ -6,3 +6,155 @@ Green: #00B172
 White: #FFF
 
 */
+
+html,
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 14px;
+    margin: 0;
+    height: 100%;
+}
+
+body {
+    background-size: cover;
+    height: 100%;
+    width: 1000px;
```

Comment:
Your website would be better if you removed `margin: 0 auto;` and changed line 21 to `width: auto;`